frontend/website/docs/faq.md

```diff
@@ -66,7 +66,30 @@ It very likely will. However, it is far less of an issue than it was a few years
 
 Coda does not support smart contracts currently. The development team is looking into smart contract models, and it is on the roadmap for future development.
 
-#### Does this protocol offer secret transactions, like in Zcash?
+#### Does Coda support shielded transactions, like in Zcash?
 
 No, Coda does not natively implement privacy features at the moment. However, privacy is key consideration for cryptocurrencies, and is also on the development roadmap.
 
+## SNARKs and Snark Workers
+
+#### If I run a Snark Worker, how do I get paid for my SNARKs that I generate?
+
+Block producers (the validators who add new blocks to the blockchain) are required to buy SNARKs from the network (or from what we call the Snarketplace) and will pay out some of their block reward as fees to the snark workers who generated SNARKs. This creates a secondary incentive mechanism in the network to reward nodes that help compress transactions.
+
+#### Are Snark Workers like Proof-of-Work (PoW) miners?
+
+The only way in which Snark workers are similar to Pow miners is that both snark proving and mining (generating hashes) are parallelizable tasks, meaning that they are optimized for GPUs. However, beyond that, producing SNARKs is not similar to PoW mining. Given the same hardware and proving algorithm, producing Snark proofs always consumes a fixed amount of resources, regardless of how many other nodes are generating SNARKs. This is unlike PoW mining algorithms where the target difficulty increases as more participants join, and the cost of mining increases as a result. In addition, SNARK research is advancing quickly, and we anticipate that SNARK proving will only become cheaper over time as hardware and algorithms get more efficient.
```

Comment:
I think we should add that, similar to PoW, in Coda you can contribute compute as a SNARK Worker to earn Codas